operations/mimir-mixin/dashboards/writes.libsonnet

```
@@ -370,56 +370,58 @@ local filename = 'mimir-writes.json';
       $._config.autoscaling.ingester.enabled,
       $.row('Ingester – autoscaling')
       .addPanel(
-        $.autoScalingActualReplicas('ingester') + { title: 'Replicas (leader zone)' } +
+        local replicaTemplateQueries = [
+          |||
+            max by (name) (
+              kube_customresource_replicatemplate_spec_replicas{%(namespace_matcher)s, name=~"%(replica_template_name)s"}
+            )
+          ||| % {
+            namespace_matcher: $.namespaceMatcher(),
+            replica_template_name: $._config.autoscaling.ingester.replica_template_name,
+          },
+          |||
+            max by (name) (
+              kube_customresource_replicatemplate_status_replicas{%(namespace_matcher)s, name=~"%(replica_template_name)s"}
+            )
+          ||| % {
+            namespace_matcher: $.namespaceMatcher(),
+            replica_template_name: $._config.autoscaling.ingester.replica_template_name,
+          },
+        ];
+
+        local replicaTemplateLegends = [
+          'Tmpl spec replicas',
+          'Tmpl status replicas',
```

[nit, non blocking] Have you considered clarify it? Don't think will change much the impact on the graph UI.

```suggestion
          'Template spec replicas',
          'Template status replicas',
```